### ISSUE TITLE
Bug: ALT text is not required in the share sheet and subsequent problems

### ISSUE THREAD

**Issue by nitricware** [Dimillian/IceCubesApp#2035]
## Environment:

 - OS: iOS 17.4.1
 
 - IceCubesApp version: 1.10.35
 

## Description

When sharing an image via the share sheet, an alt text is not required although the corresponding setting is set to ON.

Later, adding an alt text when editing that post doesn't work. Clicking on the alt button, adding text, clicking done and then clicking the alt button again reveals that the alt text is gone. Therefore with the "require alt text" setting set to ON, editing such a post is not possible.

## Related Issues

 - [X] Search that this bugs don't already exist before creating it.
